Modules/JavaScript/Prototypes-Inheritance/homework.js

```diff
@@ -0,0 +1,36 @@
+console.log(`
+  Task.
+
+    Write a myNew function that replicates all the behavior of the new operator.
+    This function should take one function parameter (the constructor), plus an unknown 
+      number of additional parameters of any type (arguments for the constructor).
+    When invoked, myNew should do everything new does and return the same object new 
+    would evaluate to, as specified below:
+    
+      var john = myNew(Person, 'John', 30) - should work the same as:
+      var john = new Person('John', 30);
+`);
+
+function Person(name, age) {
+  this.name = name;
+  this.age = age;
+}
+
+Person.prototype.introduce = function(){
+  return `My name is ${this.name} and I am ${this.age}`;
+};
+
+function myNew(classType, ...args) {
+  classType.call(this, ...args)
+  myNew.prototype.introduce = classType.prototype.introduce;
```

Comment:
your `myNew` method will work only with the specified `Person` class, it's hardcoded. The task asks for `myNew` function to act as the keyword `new`, and that means that you need to make the function reusable, starting with deleting the 25th line, since you don't know whether `classType` will have an `introduce` method in it's prototype. I suggest reading about `Object.create` when coming up with your solution